php-checks/src/main/java/org/sonar/php/checks/UselessExpressionStatementCheck.java

```diff
@@ -95,6 +100,11 @@ public void visitExpressionStatement(ExpressionStatementTree tree) {
       uselessNodes.add(tree);
     }
 
+    if (expression.is(Kind.REGULAR_STRING_LITERAL)
+      && regularStringExceptionsPatterns().anyMatch(pattern -> !pattern.matcher(expression.toString()).find())) {
```

Comment:
Rather than using `expression.toString()`, I would prefer to use the more explicit `((LiteralTree) expression).value()`.